conslogging/conslogging.go

```diff
@@ -467,11 +468,27 @@ func (cl ConsoleLogger) color(c *color.Color) *color.Color {
 	return noColor
 }
 
+var (
+	// githubRegExp Matches :2dd88e53f2e59e96ec1f9215f24a3981e5565edf+ in a target.
+	// 	Prefix containing hash may resemble: g/e/hello-world:2dd88e53f2e59e96ec1f9215f24a3981e5565edf+base
+	githubRegExp = regexp.MustCompile(`:[a-zA-Z0-9]*\\+`)
```

Comment:
Will this match any string after `:`? Branch name, etc? Do SHAs ever contain upper-case chars?

Maybe we should add a length range like `{8,}`?

Reply:
The code now properly handles branch name vs sha